applications/StructuralMechanicsApplication/python_scripts/structural_response_function_factory.py

```diff
@@ -24,6 +24,9 @@ def CreateResponseFunction(response_id, response_settings, model):
 
     elif response_type == "adjoint_local_stress":
         return structural_response.AdjointResponseFunction(response_id, response_settings, model)
+    
+    elif response_type == "adjoint_nonlinear_strain_energy":
```

Comment:
```suggestion
    elif response_type == "adjoint_non_linear_strain_energy":
```

minor, to be consistent with the rest of StructuralMechanics